contrib/runners/action_chain_runner/action_chain_runner/runner.yaml

```diff
@@ -12,3 +12,8 @@
       default: []
       description: List of tasks to skip notifications for.
       type: array
+  output_schema:
+    published:
+      type: "object"
```

Comment:
Do we need to add `additionalProperties: false` here?

Reply:
Actually yeah this might be a good place to add it. I enforce that on actions but not runners because I couldn't refactor all runners w/o some more time investment and that transition needs to happen over a couple versions since output will change. Moving `additionalProperties` here will help w/ it defaulting to `false` for now if nothing is provided.

Reply:
Never mind. Implementing this here it here led to some weirdness w/ needing to understand some jsonschema semantics are going on. Either way the net result is that additional params is set to false in the container.